packages/versioning/src/versioning.ts

```diff
@@ -923,12 +909,15 @@ export function hasDifferentReturnTypeAtVersion(
 }
 
 export function getVersionForEnumMember(program: Program, member: EnumMember): Version | undefined {
-  const parentEnum = member.enum;
-  if (!parentEnum) {
-    return undefined;
+  const globalNs = program.checker.getGlobalNamespaceType();
+  for (const [_, ns] of globalNs.namespaces) {
+    const [_, versionMap] = getVersions(program, ns);
+    const version = versionMap?.getVersionForEnumMember(member);
+    if (version) {
+      return version;
+    }
```

Comment:
@nguerrera so here we just iterate through all of the namespaces to find a version match, but this has the same drawback as the reverse lookup table did. It finds the version during compilation but returns undefined during projection (but the versionMap is NOT undefined).

Reply:
This logic was tossed.